pkgs/by-name/an/anime4k/package.nix

```diff
@@ -0,0 +1,26 @@
+{ lib
+, runCommandLocal
+, fetchFromGitHub
+}:
+
+runCommandLocal "anime4k"
```

Comment:
I prefer the mkDerivation style.

Reply:
why tho? we're just copying files from git to $out. I think we're avoiding preferlocalbuild, disabling phases etc. this way which is better

Reply:
If we are just copying files from A to B we might as well just use fetchFromGitHub

Reply:
> why tho? we're just copying files from git to $out.

I would buy such arguments years ago, before the advent of API changes on Nixpkgs fetchers that broke all `pkgs/data/fonts` and required to rewrite them in `stdenvNoCC.mkDerivation`.

Will you wait the next breakage?

Reply:
> If we are just copying files from A to B we might as well just use fetchFromGitHub

eeeee but then wouldn't we be left with garbage from the repo in nix store?

> I would buy such arguments years ago, **before** the advent of API changes on Nixpkgs fetchers that broke all `pkgs/data/fonts` and required to rewrite them in `stdenvNoCC.mkDerivation`.
> 
> Will we wait the next breakage?

wasn't aware of that, thanks

Reply:
Remade with `stdenvNoCC.mkDerivation`